src/CalculateUtil.js

```diff
@@ -0,0 +1,28 @@
+import BASIC_SEPARATOR from "./constants/basicSeparator";
+
+const CalculateUtil = {
+  parseInput(input) {
+    const separator = [...BASIC_SEPARATOR];
+    const re = /(\/\/).+(\\n)/;
+
+    if (input.match(re)) {
+      input
+        .substring(2, input.indexOf("\\"))
+        .split("")
+        .map((sep) => separator.push(sep));
+      const expression = input.substring(input.indexOf("n") + 1);
+      return { separator, expression };
+    }
+
+    const expression = input;
+    return { separator, expression };
+  },
+
+  sumExpression(separator, expression) {
+    const regex = new RegExp(`${separator.join("|")}`, "g");
```

Comment:
```suggestion
    const regex = new RegExp(separator.join("|"), "g");
```
여기는 템플릿 리터럴 문법을 사용하지 않아도 될 것 같은데 어떻게 생각하시나요??

Reply:
그렇네요 ㅎㅎ 감사합니다! 😊